components/feature/contextmenu/src/main/res/values/strings.xml

```diff
@@ -17,6 +17,8 @@
     <string name="mozac_feature_contextmenu_copy_image_location">Copy image location</string>
     <!-- Text for context menu item to save / download the image. -->
     <string name="mozac_feature_contextmenu_save_image">Save image</string>
+    <!-- Text for context menu item to save / download the video / audio. -->
+    <string name="mozac_feature_contextmenu_save_video_audio">Save video/audio</string>
```

Comment:
Doesn't it make sense to have two strings (or candidates) here? Seems weird to use "video/audio" if we know what it is. :)

Reply:
I agree.  But from testing in some cases audio is returning `HitResult.VIDEO` so it says `Save video` when its supposed to say `Save audio`.  Looks like when the audio is being played the site might use the video player.